packages/data-grid/src/table/Table.jsx

```diff
@@ -58,6 +58,10 @@ const propTypes = {
     /** Descibes the column being sorted using the column dataKey  */
     sortDataKey: string,
   }),
+  /** Boolean to determine whether to display loading state */
+  isLoading: bool,
```

Comment:
I honestly prefer prefixes like `is` or `has` for boolean props, but the convention was already established to not use them, so refactoring `isLoading` to `loading` would increase consistency with other design-system components.